erpnext/projects/report/billing_summary.py

```diff
@@ -6,6 +6,7 @@
 import frappe
 from frappe import _
 from frappe.utils import time_diff_in_hours, flt
+from frappe.model.meta import get_field_precision
```

Comment:
get_field_precision imported but not used

Reply:
removed